65black/StackOverflow-lite
# [feature #160160178] Get User Answers

## What does this PR do?
Adds logic for the` GET /api/v1/users/answers` endpoint.
The user must be signed in to access this endpoint.

If the user is signed in, then hitting this endpoint will return an array of all the answers ever posted by the user.

### Description of Task to be completed?
Have the following endpoints working:

> GET /api/v1/users/answers

### How should this be manually tested?
After cloning the repo, cd into it and run `npm run start-dev`. Follow these steps:

 - Open postman and hit the login endpoint first to get your auth token.
 - Add the token to your request header and hit `/api/v1/users/answers`.
 - You should get a response with an array of answers.

### Screenshots
![scr](https://user-images.githubusercontent.com/36507972/45268756-862def80-b479-11e8-8e80-0574476b21ca.JPG)


### What are the relevant pivotal tracker stories?
#160160178
